### PR TITLE
Changing appearance of card hands

### PR DESCRIPTION
**Pull request by Dingo-Dongler**
## About The Pull Request

This makes card hands have an overlay of the top card (putting a new card in a hand puts it on top). This makes seeing what you hold, and some card games like gin rummy easier to play.

cards

Pictured is 1 card (no hand), 2 cards, 3 cards and 4 cards in hands

## Why It's Good For The Game
Cards should work better than they do.

## Changelog
:cl:
tweak: Hands of cards show their top card.
/:cl:

### CODE REVIEW

**Comment by Crossedfall** [comment]
Would it be possible to provide a screenshot showing the difference in your description?

**Comment by Dingo-Dongler** [comment]
Donezo, I was considering adding a facedown card for hands of 4+ but I couldn't really fit it in.

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -886,6 +875,20 @@
...
+
+	var/k
+	if(overlay_cards == 2)
+		k = 1
+	else
+		k = overlay_cards - 2
+
+	for(var/i = k; i <= overlay_cards; i++)
+		var/card_overlay = image(icon=src.icon,icon_state="sc_[currenthand[i]]_[deckstyle]",pixel_x=(1-i+k)*3,pixel_y=(1-i+k)*3)
+		add_overlay(card_overlay)
```
Comment (Crossedfall) [inline code/game/objects/items/toys.dm]:
```suggestion
	var/k = overlay_cards == 2 ? 1 : overlay_cards - 2
	for(var/i = k; i <= overlay_cards; i++)
		var/card_overlay = image(icon=src.icon,icon_state="sc_[currenthand[i]]_[deckstyle]",pixel_x=(1-i+k)*3,pixel_y=(1-i+k)*3)
		add_overlay(card_overlay)
```

From: https://github.com/tgstation/tgstation/pull/50357#discussion_r403068216

**Comment by qwertyquerty** [comment]
Fix conflicts

**Comment by Dingo-Dongler** [comment]
No clue what that change to changelog is

**Comment by Dingo-Dongler** [comment]
Undid that, should be good to go

**Pull request merged**